awspub/image.py

```diff
@@ -502,6 +457,55 @@ def create(self) -> Dict[str, _ImageInfo]:
 
         return images
 
+    def _register_snapshots(
```

Comment:
commit 6a141ca311554e7e67c1473a49836eaf057a498 does name this function wrongly and then the follow up commit does rename again. please fix the commits so each commit is correct